### PR TITLE
[FilteringWorkspaceL] remove signals duplication

### PR DESCRIPTION
**Pull request by mathildemerle**
Working on https://github.com/medInria/medInria-public/issues/593 i found some duplication of signals. 

:m:

### CODE REVIEW

File: src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp [hunk 1]
```diff
@@ -35,8 +35,6 @@ class medFilteringWorkspaceLPrivate
 medFilteringWorkspaceL::medFilteringWorkspaceL(QWidget *parent)
     : medSelectorWorkspace (parent, staticName()), d(new medFilteringWorkspaceLPrivate)
 {
-    connect(this->tabbedViewContainers(), SIGNAL(containersSelectedChanged()),
```
Comment (Florent2305) [inline src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp:38]: I don't understand why it need to be removed

Comment (mathildemerle) [reply src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp:38]: inputChanged is already called in medFilteringWorkspaceL::changeToolBoxInput() below

Comment (mathildemerle) [reply src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp:38]: And this line emits inputChanged even if the output image is changed. We do not want that. That's why in medFilteringWorkspaceL::setupTabbedViewContainer() we choose what we want to do with either the input or the output container

Comment (Florent2305) [reply src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp:38]: ok

File: src/plugins/legacy/itkFilters/itkFiltersToolBox.cpp [hunk 2]
```diff
@@ -496,6 +496,7 @@ void itkFiltersToolBox::clear()
     d->maxColor = Qt::white;
     d->thresholdColor = Qt::black;
     d->minValueImage = d->maxValueImage = 0.;
+    d->process = nullptr;
```
[the remark this answers is not in the document]
Comment (mathildemerle) [reply src/plugins/legacy/itkFilters/itkFiltersToolBox.cpp:499]: this is a smartpointer

File: src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp (hunk 1 above)
Comment (Florent2305) [reply src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp:38]: Indeed :)

**Pull request merged**